assets/scss/02-molecules/_labelled-list.scss

```diff
@@ -1,4 +1,9 @@
 .ma__labelled-list {
+  margin-top: 30px !important;

```

Comment:
@clairesunstudio  --> do we want to consistently use rem for margins?

Reply:
I think we haven't been doing margin/padding consistently, which we should... we should standardize it to use either rem or em (inherit from the parent, which could be more modular)

Reply:
For now maybe let's try to use rem for margin/padding till we make a decision where to use em